### PR TITLE
Ana/default to transaction denom for fee when 1 amount

### LINKED ISSUE #260: SendTx fee denom should default to denom of transaction
For a smoother UX, I'd suggest the fee denom defaults to whatever is being sent.

Specifically, it should default to the first (and possibly only) denom in the SendTx.

Demonstrated below:

image
image

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #260

### CODE REVIEW

File: components/ActionModals/TableInvoice.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <FormGroup
-      v-if="getDenoms.length > 1"
+      v-if="getDenoms.length > 1 && amounts.length > 1"
```
Comment (faboweb) [inline components/ActionModals/TableInvoice.vue:4]: nope. you should be able to pay with any of the available fees

Comment (Bitcoinera) [reply components/ActionModals/TableInvoice.vue:4]: If there is only one amount (one denom being transacted), it makes sense to default to that denom to pay the fee. Ah yeah, but you could still be able to choose.

OK, changing this

**Comment by Bitcoinera** [comment]
Now it is good

File: components/ActionModals/TableInvoice.vue [hunk 2]
```diff
@@ -82,10 +82,17 @@ export default {
...
   },
-  data: () => ({
-    feeDenom: network.stakingDenom,
-  }),
   computed: {
+    feeDenom: {
+      get() {
+        return this.amounts.length > 1
+          ? network.stakingDenom
+          : this.amounts[0].denom
```
Comment (faboweb) [inline components/ActionModals/TableInvoice.vue:90]: this assumes the fee for this denom is specified

Comment (Bitcoinera) [reply components/ActionModals/TableInvoice.vue:90]: ah, true

File: components/ActionModals/TableInvoice.vue [hunk 3]
```diff
@@ -82,10 +82,21 @@ export default {
...
-  data: () => ({
-    feeDenom: network.stakingDenom,
-  }),
   computed: {
+    feeDenom: {
+      get() {
+        // default to a denom if there is the only one being transacted and it has feeOptions specified
+        const transactionDenom =
+          this.amounts.length === 1 &&
+          this.fees.find(({ denom }) => this.amounts[0].denom)
```
Comment (faboweb) [inline components/ActionModals/TableInvoice.vue:91]:
```suggestion
          this.fees.find(({ denom }) => denom === this.amounts[0].denom)
```

File: components/ActionModals/TableInvoice.vue [hunk 4]
```diff
@@ -82,10 +82,21 @@ export default {
...
-  data: () => ({
-    feeDenom: network.stakingDenom,
-  }),
   computed: {
+    feeDenom: {
+      get() {
+        // default to a denom if there is the only one being transacted and it has feeOptions specified
+        const transactionDenom =
+          this.amounts.length === 1 &&
+          this.fees.find(({ denom }) => denom === this.amounts[0].denom)
```
Comment (faboweb) [inline components/ActionModals/TableInvoice.vue:91]: do we need to put this term in "()" or does && bind enough?

Comment (Bitcoinera) [reply components/ActionModals/TableInvoice.vue:91]: I used the linting and this stayed the same

**Pull request merged**